.github/workflows/firebase-merge.yml

```
@@ -25,4 +25,4 @@ jobs:
         with:
           args: deploy --only functions
         env:
-          FIREBASE_TOKEN: ${{ secrets.FIREBASE_TOKEN }}
+          FIREBASE_TOKEN: ${{ secrets.GCP_SA_KEY }}
```

I believe the config should be changed like this

```suggestion
          firebaseServiceAccount: '${{ secrets.GCP_SA_KEY }}'
          FIREBASE_TOKEN: ${{ secrets.FIREBASE_TOKEN }}
```